src/extension/tab/hook.ts

```diff
@@ -91,12 +91,6 @@ window.onbeforeunload = () => {
   tab.send({ type: "disconnectFromDevtools" });
 };
 
-window.addEventListener("load", () => {
-  if (hook.ApolloClient) {
```

Comment:
This was almost never true, but with the changes in this PR, we don't need it anyways since we will register the client immediately if its available when this content script loads, or register it when the client is assigned on the window.